tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -43,8 +43,7 @@ public class CrudTestRunner
             "authenticate",
             "saslStart",
             "saslContinue",
-            "getnonce",
-            "find"
```

Comment:
If we don't capture any Find commands we can't assert against them!

Unfortunately, we also end up capturing some extra Find commands that the JSON tests aren't expecting. See RemoveExtraFindCommandsFromActualEvents.

Reply:
Acknowledged.